pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -111,10 +111,9 @@ type SchedulingQueue interface {
 	Done(types.UID)
 	Update(logger klog.Logger, oldPod, newPod *v1.Pod)
 	Delete(pod *v1.Pod)
-	// TODO(sanposhiho): move all PreEnqueueCheck to Requeue and delete it from this parameter eventually.
-	// Some PreEnqueueCheck include event filtering logic based on some in-tree plugins
-	// and it affect badly to other plugins.
```

Comment:
I was planning to remove all PreCheck. But, actually we cannot and don't have to remove the following preCheck:
https://github.com/kubernetes/kubernetes/blob/95956671d8da7783a726133709b8085f56dda052/pkg/scheduler/schedule_one.go#L366